src/main/kotlin/no/nav/familie/ef/mottak/mapper/ArkiverDokumentRequestMapper.kt

```diff
@@ -35,15 +34,15 @@ object ArkiverDokumentRequestMapper {
     fun fromEttersending(ettersending: Ettersending,
                          vedlegg: List<EttersendingVedlegg>): ArkiverDokumentRequest {
 
-        val ettersendingDto = EttersendingMapper.toDto<EttersendingDto>(ettersending)
+        val stønadType = StønadType.valueOf(ettersending.stønadType)
 
-        val hovedDokumentVarianter = lagHoveddokumentvarianterForEttersending(ettersendingDto.stønadType, ettersending)
+        val hovedDokumentVarianter = lagHoveddokumentvarianterForEttersending(stønadType, ettersending)
 
-        return ArkiverDokumentRequest(ettersendingDto.fnr,
+        return ArkiverDokumentRequest(ettersending.fnr,
                                       false,
                                       hovedDokumentVarianter,
                                       mapEttersendingVedlegg(vedlegg,
-                                                             ettersendingDto.stønadType))
+                                                             stønadType))
```

Comment:
Dette fungerer fordi en Ettersending kun kan ha en stønadstype. Trenger vi validering på at vedleggene er for samme stønadstype?

Reply:
Høres jo ut som en god idé :)